rust/protocol/Cargo.toml

```diff
@@ -24,7 +24,7 @@ prost = "0.9"
 rand = "0.7.3"
 sha2 = "0.9"
 subtle = "2.2.3"
-x25519-dalek = "1.0"
+x25519-dalek = "1.1, <1.2"
```

Comment:
Nitpick: this can be written as `"~1.1"`.

Reply:
Fixed.